airwallex/src/main/java/com/airwallex/android/view/AddPaymentMethodActivity.kt

```diff
@@ -207,8 +214,11 @@ internal class AddPaymentMethodActivity : AirwallexCheckoutBaseActivity() {
         if (session is AirwallexPaymentSession && session.customerId != null) {
             viewBinding.saveCardWidget.visibility = View.VISIBLE
             viewBinding.swSaveCard.setOnCheckedChangeListener { _, isChecked ->
-                if (currentBrand == CardBrand.UnionPay && isChecked) {
```

Comment:
The logic here is not exactly the same as what it was before. You could fallback to else branch with isChecked being `true` but currentBrand being other things. But now, isChecked has to be false in order to fallback to else.

Reply:
Good pickup! Just fixed.